clang/test/SemaSYCL/sycl-kernel-entry-point-attr-kernel-name-pch.cpp

```diff
@@ -15,6 +15,10 @@
 #--- pch.h
 template<int> struct KN;
 
+// A generic kernel launch function.
+template<typename KN, typename... Ts>
+void sycl_kernel_launch(const char *, Ts...) {}
```

Comment:
Or at least... do we intend the library to support these?  Maybe we should have an Inputs header for all of these...